### PR TITLE
Remove the Pokemon Showdown Day link

### PR DESCRIPTION
**Pull request by SSlayer9**
https://www.smogon.com/forums/threads/bug-reports-v4-read-original-post-before-posting.3663703/post-8481522 this. (Please edit in a link crediting whoever made it if you want to).

### CODE REVIEW

File: js/storage.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ Storage.bg = {
 				break;
 			case 'psday':
 				hues = ["24.705882352941174,25.37313432835821%", "260.4651162790697,59.44700460829492%", "165.3191489361702,46.07843137254901%", "16.363636363636367,42.63565891472869%", "259.04761904761904,34.05405405405405%", "24.705882352941174,25.37313432835821%"];
-				attrib = '<a href="https://pokemonshowdown.com/news/216" target="_blank" class="subtle">Pokemon Showdown Day!</a>';
+				attrib = 'Pokemon Showdown Day Background!';
```
Comment (Zarel) [inline js/storage.js:123]: Should probably credit the artist. I think it was LiD? We'll have to ask around.

Comment (scheibo) [reply js/storage.js:123]: > dont use my full name ppl are still curious to where i am so like i dont want that. Just LifeisDANK nothing else needed I have no real online portfolio anymore.

Was what LiD responded with when you asked her about this, check #639

Comment (Zarel) [reply js/storage.js:123]: Oh yeah I remember trying to figure out how to best implement that.

Comment (Zarel) [reply js/storage.js:123]:
```suggestion
				attrib = 'Pokémon Showdown Day background <small>by LifeisDANK</small>';
```

Comment (KrisXV) [reply js/storage.js:123]: `Pok&eacute;mon` with an escaped accent would be better, no?

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**